batchflow/tests/notebooks/torch_benchmark.ipynb

```diff
@@ -0,0 +1,4784 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            'K_1357': Block(inputs=inputs, layout='K', channels='same', kernel_size=(1, 3, 5, 7), dilation=1),
</pre><p>Not clear naming</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>